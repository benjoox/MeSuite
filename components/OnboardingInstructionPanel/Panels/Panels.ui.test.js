import React from 'react'
import renderer from 'react-test-renderer'
import Panel from '.'

it('renders correctly', () => {
    const tree = renderer.create(<Panel message="this is a message" />).toJSON()
    expect(tree).toMatchSnapshot()
})
