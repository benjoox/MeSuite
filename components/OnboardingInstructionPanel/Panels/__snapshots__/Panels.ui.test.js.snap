// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Panel section  renders correctly 1`] = `
<div
  className="justify-content-center column row"
  style={
    Object {
      "margin": "1rem",
    }
  }
>
  <div
    className="jumbotron"
    style={
      Object {
        "textAlign": "center",
      }
    }
  >
    <p
      style={
        Object {
          "height": "90px",
        }
      }
    >
      this is a message
    </p>
    <form>
      <button
        className="btn btn-outline-dark"
        disabled={false}
        onClick={[Function]}
        type="button"
      >
        Upload
      </button>
      <input
        accept=".csv"
        hidden={true}
        onChange={[Function]}
        type="file"
      />
    </form>
  </div>
</div>
`;
